package.json

```diff
@@ -18,7 +18,7 @@
     "pull-stream": "^3.6.14",
     "private-group-spec": "^2.1.1",
     "ssb-bfe": "^3.7.0",
-    "ssb-keyring": "^4.1.0",
+    "ssb-keyring": "https://gitlab.com/powersource/ssb-keyring/-/archive/multiple-keys/ssb-keyring-multiple-keys.tar.gz",
```

Comment:
https://gitlab.com/ahau/lib/ssb-keyring/-/merge_requests/22

Reply:
merged. can someone publish a new major or add me as a maintainer? @arj03 @staltz @mixmix

Reply:
I tried, but it says forbidden :(

Reply:
I made a new major version of keyring now.